### ISSUE TITLE
Remove dependency on live hbase when using HBaseSnapshotInputFormat

### ISSUE THREAD

**Issue by jerryjch** [JanusGraph/janusgraph#838]
This is a follow-up issue after #656.

The current JanusGraph input format base implementation needs to go to a live backend to get schema data. This is ok, particularly for input formats that operate on live backend, e.g. CassandraInputFormat and HBaseInputFormat, because we scan the table on live backend.
But HBaseSnapshotInputFormat works on a snapshot, which is static data on a file system. For example, it can be an exported snapshot stored on S3 without a live HBase running.  It is self-contained with all the table data for a graph.  We should be able to get the schema data from the static snapshot in addition to the vertices and edge data.  Then we will be able to work on a snapshot independently, the way it should.
